test/plugin/test_compressable.rb

```diff
@@ -0,0 +1,71 @@
+require_relative '../helper'
+require 'fluent/plugin/compressable'
+
+class CompressableTest < Test::Unit::TestCase
+  include Fluent::Plugin::Compressable
+
+  sub_test_case '#compress' do
+    setup do
+      @src = 'text data for compressing' * 5
+      @gzipped_src = compress(@src)
+    end
+
+    test 'compress data' do
+      str = "this is a sample text" * 10
+      compressed_str = "\x1F\x8B\b\x00Y;\xC6W\x00\x03+\xC9\xC8,V\x00\xA2D\x85\xE2\xC4\xDC\x82\x9CT\x85\x92\xD4\x8A\x92\x92\xA1,\b\x00\xEF}\x9C\xF1\xD2\x00\x00\x00"
+      assert_not_equal compressed_str, compress(str)
+      assert_not_equal @gzipped_src, @src
+    end
```

Comment:
It's better to have assertion to compare the length of src and gzipped... because it's about "compression".